app/views/ukandforeignproperty/UkAndForeignPropertyClaimPropertyIncomeAllowanceOrExpensesView.scala.html

```diff
@@ -30,44 +30,44 @@
 
 @(form: Form[_], taxYear: Int, mode: Mode, individualOrAgent: String)(implicit request: Request[_], messages: Messages)
 
-@layout(pageTitle = title(form, messages("claimPropertyIncomeAllowanceOrExpenses.title"))) {
+@layout(pageTitle = title(form, messages("ukAndForeignPropertyClaimPropertyIncomeAllowanceOrExpenses.title"))) {
```

Comment:
@iyadi The message ukAndForeignPropertyClaimPropertyIncomeAllowanceOrExpenses.title is missing in the messages file